vfox-restServer/src/main/java/com/vfoexchange/restServer/controller/UserController.java

```diff
@@ -235,4 +236,24 @@ public ResponseDTO userVerification(@RequestBody UserDTO userDTO) {
         return responseEntity;
     }
 
+    /*
+    Method for capturing enquiring user contact details
+    */
+    @RequestMapping(value = "/enquiring/contact/details", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public ResponseDTO captureContactDetails(@RequestBody EnquiringUserDTO enquiringUserDTO) {
+        ResponseDTO resp = new ResponseDTO();
+        try {
+            userService.captureContactDetails(enquiringUserDTO);
+            resp.setCode("200");
+            resp.setMsg("Enquiring user's contact details captured successfully");
+            LOGGER.info("Enquiring user's contact details captured successfully");
```

Comment:
Add enquiringUserDTO.toString() to LOGGER message. Otherwise how would we know who this message is for?